### PR TITLE
Filipino Translation - 1 202 translated words

### PR DESCRIPTION
**Pull request by mind-blast**
Translating bundling.asc - 1202 translated words.
Note : Code text and computer code generated text are not included as translated words.

### CODE REVIEW

File: book/07-git-tools/sections/bundling.asc [hunk 1]
```diff
@@ -1,20 +1,20 @@
 [[_bundling]]
-=== Bundling
+=== Pagbibigkis
 
-Though we've covered the common ways to transfer Git data over a network (HTTP, SSH, etc), there is actually one more way to do so that is not commonly used but can actually be quite useful.
+Kahit na tinalakay natin ang pangkaraniwang paraan sa paglipat sa Git na datos gamit ang network (HTTP, SSH, etc), may isa talaga isang paraan na gawin ang hindi karaniwang ginamit ngunit maaaring talagang maging kapaki-pakinabang.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/bundling.asc]: `mayroon talagang isa pang paraan upang gawin iyon na hindi karaniwang`

File: book/07-git-tools/sections/bundling.asc [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-Git is capable of ``bundling'' its data into a single file.
-This can be useful in various scenarios.
-Maybe your network is down and you want to send changes to your co-workers.
-Perhaps you're working somewhere offsite and don't have access to the local network for security reasons.
-Maybe your wireless/ethernet card just broke.
-Maybe you don't have access to a shared server for the moment, you want to email someone updates and you don't want to transfer 40 commits via `format-patch`.
+Ang Git ay may kakayahan sa ``pagbibigkis'' sa datos nito sa isang solong file.
+Ito may maaaring magagamit sa iba't ibang sitwasyon.
+Baka ang iyong network ay nasira at gusto mo na pagpasa ng mga pagbabago sa iyong mga katrabaho.
+Marahil ikaw ay nagtratrabaho na kung saan offsite at wala kang pahintulot sa lokal na network para sa mga rason sa seguridad.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/bundling.asc]: - `nagtatrabaho`
- `kung saang`

File: book/07-git-tools/sections/bundling.asc [hunk 3]
```diff
@@ -44,14 +44,14 @@ Writing objects: 100% (6/6), 441 bytes, done.
 Total 6 (delta 0), reused 0 (delta 0)
 ----
 
-Now you have a file named `repo.bundle` that has all the data needed to re-create the repository's `master` branch.
-With the `bundle` command you need to list out every reference or specific range of commits that you want to be included.
-If you intend for this to be cloned somewhere else, you should add HEAD as a reference as well as we've done here.
+Mayroon ka na ngayong isang file na tinatawag na `repo.bundle` na merong lahat ng datos na kinakailangan mo maggawa muli ang `master` branch ng repositoryo.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/bundling.asc]: `master` na branch

File: book/07-git-tools/sections/bundling.asc [hunk 4]
```diff
@@ -78,14 +78,14 @@ c99cf5b fourth commit - second repo
 b1ec324 first commit
 ----
 
-First we need to determine the range of commits we want to include in the bundle.
-Unlike the network protocols which figure out the minimum set of data to transfer over the network for us, we'll have to figure this out manually.
- Now, you could just do the same thing and bundle the entire repository, which will work, but it's better to just bundle up the difference - just the three commits we just made locally.
+Una kailangan nating matukoy ang layo ng mga commit na gusto nating isali sa bundle.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/bundling.asc]: bundle to bugkos

File: book/07-git-tools/sections/bundling.asc [hunk 5]
```diff
@@ -95,8 +95,8 @@ c99cf5b fourth commit - second repo
 7011d3d third commit - second repo
 ----
 
-So now that we have the list of commits we want to include in the bundle, let's bundle them up.
-We do that with the `git bundle create` command, giving it a filename we want our bundle to be and the range of commits we want to go into it.
+Sa ngayon meron na tayong listahan sa mga commit na gusto nating isali sa bugkos, kaya ibugkos na natin sila.
+Ginagawa natin ito sa pamamagitan ng `git bundle create`na utos, magbigay ka ng isang filename na gusto mong ibugkos at ang lawak ng mga commit na gusto nating puntahan.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/bundling.asc]: space between `git bundle create` and utos

File: book/07-git-tools/sections/credentials.asc [hunk 6]
```diff
@@ -1,29 +1,29 @@
...
-  This method stores the credentials on disk, and they never expire, but they're encrypted with the same system that stores HTTPS certificates and Safari auto-fills.
-* If you're using Windows, you can install a helper called ``Git Credential Manager for Windows.''
-  This is similar to the ``osxkeychain'' helper described above, but uses the Windows Credential Store to control sensitive information.
-  It can be found at https://github.com/Microsoft/Git-Credential-Manager-for-Windows[].
-
-You can choose one of these methods by setting a Git configuration value:
+Kung ikaw ay gagamit ng SSH transport para sa pagkonekta ng mga remote, ito ay posible para sa iyo na magkaroon ng isang susi na walang isang passphrase, na kung saan ay nagpapahintulot sa iyo ng ligtas na paglipat ng datos na walang pag-type ng iyong username at password.
+Gayunpaman, ito ay hindi posible sa HTTP na mga protocol – sa bawat koneksyon ay nangangailangan ng isang username at password.
+Ito ay naging mas mahirap para sa mga sistema na may dalawang-factor sa pagpapatotohanan, na kung saan ang token na iyong ginamit para sa password ay randomly na nabuo at unpronounceable.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/credentials.asc]: - `randomly` to sapalaran
- `unpronounceable` to `hindi mabibigkas`

File: book/07-git-tools/sections/credentials.asc [hunk 7]
```diff
@@ -1,29 +1,29 @@
...
+
+Sa kabutihang-palad, ang Git ay merong maraming mga kredensyal na sistema na maari makakatulong para dito.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/credentials.asc]: `maari` to `maaari`

File: book/07-git-tools/sections/credentials.asc [hunk 8]
```diff
@@ -1,29 +1,29 @@
...
+Ang Git ay mayroong ilang mga opsyon na makikita sa ibinigay na kahon:
+
+* Ang default ay hindi na-cache sa lahat.
+  Sa bawat koneskyon ay mayroong lumalabas para sayong username at password.
+* Ang ``cache'' na mode ay nagpapanatili ng mga kredensyal sa memorya para sa isang tiyak na oras.
+  Wala sa mga password ang na-imbak sa disk, at sila ay nalinis mula sa cache pagkatapos ng 15 minuto.
+* Ang ``store'' na mode ay sine-save ang mga kredensyal sa isang plain-teksyo sa disk, at sila ay hindi pa na-expire.
+  Nangangahulugan ito na hanggang binago mo ang iyong password para sa Git host, ay hindi ka na kailanman kailangan i-type ang iyong kredensyal muli.
+  Ang downside sa lapitan na ang iyong password ay naka-imbak sa cleartext sa isang plain file sa iyong home na direktoryo.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/credentials.asc]: `downside` to `kakulangan`

File: book/07-git-tools/sections/credentials.asc [hunk 9]
```diff
@@ -1,29 +1,29 @@
...
+* Kung ikaw ay gumagamit ng Mac, ang Git ay mayroong isang ``osxkeychain'' na mode, na may mga cache na mga kredensyal sa isang ligtas na keychain na naka-attach sa sistema ng iyong account.
+  Ang paraang ito ay nag-iimbak ng mga kredensyal sa disk, at hindi sila mag-expire, ngunit sila ay naka-encryprt na may parehong sistema na nag-iimbak ng HTTPS na mga sertipiko at Safari awtomatikong nagpupunan.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/credentials.asc]: - `naka-encrypt` 
- `ang Safari ay awtomatikong nagpupuna`

File: book/07-git-tools/sections/credentials.asc [hunk 10]
```diff
@@ -1,29 +1,29 @@
...
+* Kung ikaw ay gumagamit ng Windows, maaari kang mag-install ng tumutulong na nagngangalang ``Git Credential Manager for Windows.''
+  Ito ay katulad sa ``osxkeychain'' na tumutulong helper na naglalarawan sa itaas, ngunit gumagamit ng Windows Credential Store upang i-kontrol ang sensitibong impormasyon.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/credentials.asc]: `na tumutulong sa paglalarawan sa itaas`

File: book/07-git-tools/sections/credentials.asc [hunk 11]
```diff
@@ -1,29 +1,29 @@
...
+  Ito ay maaaring matagpuan sa https://github.com/Microsoft/Git-Credential-Manager-for-Windows[].
+
+Maaari kang pumili sa isa sa mga pamaraan ng pagtatakda ng Git na pagsasaayos na halaga:
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/credentials.asc]: `Maaari kang pumili sa isa sa mga pamamaraan ng pagtatakda ng halaga sa kompigurasyon ng Git`

**Review by josteinfrancois — COMMENTED** [review]
Hello @robin-ho . I have finished reviewing your work. Please see the comments. Thanks.

File: book/07-git-tools/sections/bundling.asc (hunk 1 above)
Comment (mind-blast) [reply book/07-git-tools/sections/bundling.asc]: done change

File: book/07-git-tools/sections/bundling.asc (hunk 2 above)
Comment (mind-blast) [reply book/07-git-tools/sections/bundling.asc]: done change

File: book/07-git-tools/sections/bundling.asc (hunk 3 above)
Comment (mind-blast) [reply book/07-git-tools/sections/bundling.asc]: done change

File: book/07-git-tools/sections/bundling.asc (hunk 4 above)
Comment (mind-blast) [reply book/07-git-tools/sections/bundling.asc]: done change

File: book/07-git-tools/sections/bundling.asc (hunk 5 above)
Comment (mind-blast) [reply book/07-git-tools/sections/bundling.asc]: done change

File: book/07-git-tools/sections/credentials.asc (hunk 6 above)
Comment (mind-blast) [reply book/07-git-tools/sections/credentials.asc]: done change

File: book/07-git-tools/sections/credentials.asc (hunk 7 above)
Comment (mind-blast) [reply book/07-git-tools/sections/credentials.asc]: done change

File: book/07-git-tools/sections/credentials.asc (hunk 8 above)
Comment (mind-blast) [reply book/07-git-tools/sections/credentials.asc]: done change

File: book/07-git-tools/sections/credentials.asc (hunk 9 above)
Comment (mind-blast) [reply book/07-git-tools/sections/credentials.asc]: done change

File: book/07-git-tools/sections/credentials.asc (hunk 10 above)
Comment (mind-blast) [reply book/07-git-tools/sections/credentials.asc]: done change

File: book/07-git-tools/sections/credentials.asc (hunk 11 above)
Comment (mind-blast) [reply book/07-git-tools/sections/credentials.asc]: done change

**Comment by mind-blast** [comment]
done changed all suggested words. Thank you for the proof read.

**Pull request merged**